<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ProjectsWindow</class>
 <widget class="QWidget" name="ProjectsWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>804</width>
    <height>556</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QWidget" name="verticalLayoutWidget">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>90</y>
     <width>411</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="mainLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout">
      <item>
       <widget class="QGraphicsView" name="burndownGraphicsView"/>
      </item>
      <item>
       <widget class="QGraphicsView" name="DailyGraphicsView"/>
      </item>
     </layout>
    </item>
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_2">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_8">
          <item>
           <widget class="QLabel" name="label_13">
            <property name="text">
             <string>Start date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="startDateLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_7">
          <item>
           <widget class="QLabel" name="label_8">
            <property name="text">
             <string>End Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="endDateLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_6">
          <item>
           <widget class="QLabel" name="label_7">
            <property name="text">
             <string>B. Schedule (days)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="behindScheduleDaysLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_5">
          <item>
           <widget class="QLabel" name="label_6">
            <property name="text">
             <string>B. Schedule (work)</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="behinScheduleWorkLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_4">
          <item>
           <widget class="QLabel" name="label_5">
            <property name="text">
             <string>Expected End Date</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="expectedEndDateLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_10">
          <item>
           <widget class="QLabel" name="label_18">
            <property name="text">
             <string>Days Gone</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="daysGoneLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_3">
          <item>
           <widget class="QLabel" name="label">
            <property name="text">
             <string>Days Remaining</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="daysRemainingLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_21">
          <item>
           <widget class="QLabel" name="label_40">
            <property name="text">
             <string>Daily Reward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="dailyRewardLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_11">
          <item>
           <widget class="QLabel" name="label_20">
            <property name="text">
             <string>Chain Multiplier</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="chainMultiplierLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_9">
          <item>
           <widget class="QLabel" name="label_15">
            <property name="text">
             <string>Max Daily Reward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="maxDailyRewardLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_20">
          <item>
           <widget class="QLabel" name="label_37">
            <property name="text">
             <string>Work Amount</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="workAmountLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_19">
          <item>
           <widget class="QLabel" name="label_36">
            <property name="text">
             <string>Work Done</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="workDoneLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_22">
          <item>
           <widget class="QLabel" name="label_42">
            <property name="text">
             <string>Work Remaining</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="workRemainingLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_18">
          <item>
           <widget class="QLabel" name="label_33">
            <property name="text">
             <string>Start Daily Work</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="startDailyWorkLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_17">
          <item>
           <widget class="QLabel" name="label_32">
            <property name="text">
             <string>Cur. Daily Work</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentDailyWorkLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_16">
          <item>
           <widget class="QLabel" name="label_30">
            <property name="text">
             <string>Req. Daily Work</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="requiredDailyWorkLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_15">
          <item>
           <widget class="QLabel" name="label_28">
            <property name="text">
             <string>Cur Daily Reward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentDailyRewardLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_12">
          <item>
           <widget class="QLabel" name="label_3">
            <property name="text">
             <string>Cur. Chain Length</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentChainLengthLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_14">
          <item>
           <widget class="QLabel" name="label_26">
            <property name="text">
             <string>Cur. Chain Mult.</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="currentChainMultiplierLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
        <item>
         <layout class="QHBoxLayout" name="horizontalLayout_13">
          <item>
           <widget class="QLabel" name="label_24">
            <property name="text">
             <string>Total Reward</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QLabel" name="totalRewardLabel">
            <property name="text">
             <string>TextLabel</string>
            </property>
           </widget>
          </item>
         </layout>
        </item>
       </layout>
      </item>
     </layout>
    </item>
   </layout>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_2">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>90</y>
     <width>181</width>
     <height>271</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="projectsLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_3">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>10</y>
     <width>181</width>
     <height>76</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="entryLayout">
    <item>
     <layout class="QHBoxLayout" name="horizontalLayout_23">
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QLabel" name="label_2">
          <property name="text">
           <string>Date</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="label_4">
          <property name="text">
           <string>Work Done</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QVBoxLayout" name="verticalLayout_5">
        <item>
         <widget class="QLineEdit" name="dateLineEdit"/>
        </item>
        <item>
         <widget class="QLineEdit" name="workDoneLineEdit"/>
        </item>
       </layout>
      </item>
     </layout>
    </item>
    <item>
     <widget class="QPushButton" name="submitEntryButton">
      <property name="text">
       <string>Submit</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QProgressBar" name="progressBar">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>60</y>
     <width>411</width>
     <height>23</height>
    </rect>
   </property>
   <property name="value">
    <number>0</number>
   </property>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_6">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>90</y>
     <width>181</width>
     <height>301</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="subprojectsLayout"/>
  </widget>
  <widget class="QWidget" name="verticalLayoutWidget_7">
   <property name="geometry">
    <rect>
     <x>200</x>
     <y>10</y>
     <width>411</width>
     <height>41</height>
    </rect>
   </property>
   <layout class="QVBoxLayout" name="verticalLayout_3">
    <item>
     <widget class="QLabel" name="nameLabel">
      <property name="text">
       <string>Project Name</string>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QLabel" name="descriptionLabel">
      <property name="text">
       <string>Project Descroption</string>
      </property>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QPushButton" name="createNewProjectButton">
   <property name="geometry">
    <rect>
     <x>620</x>
     <y>370</y>
     <width>181</width>
     <height>18</height>
    </rect>
   </property>
   <property name="text">
    <string>Create New Project</string>
   </property>
  </widget>
  <widget class="QPushButton" name="mainMenuButton">
   <property name="geometry">
    <rect>
     <x>10</x>
     <y>10</y>
     <width>181</width>
     <height>71</height>
    </rect>
   </property>
   <property name="text">
    <string>Back To Main Menu</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
